suite/auto-sync/README.md

```diff
@@ -129,9 +148,66 @@ Documentation about the `.inc` file generation is in the [llvm-capstone](https:/
 
 **Formatting**
 
-- If you make changes to the `CppTranslator` please format the files with `black`
+- If you make changes to the `CppTranslator` please format the files with `black` and `usort`
   ```
-  source ./.venv/bin/activate
-  pip3 install black
-  python3 -m black --line-length=120 CppTranslator/*/*.py
+  pip3 install black usort
+  python3.11 -m usort format src/autosync
```

Comment:
Is there any reason to hardcode `python3.11` here? Maybe generic `python3` is enough?